app/controllers/templates_controller.rb

```diff
@@ -59,4 +66,10 @@ def destroy
   def template_params
     params.require(:template).permit(:name, :question_ids => [])
   end
+
+  def check_admin
```

Comment:
I add this because our admin is user too, so he still have access to templates. As i understand only interviewer has acces to templates. So i check if user admin and close access to templates for him 
def check_admin
    if current_user.admin? 
      redirect_to authenticated_root_path, notice: 'Access Denied' 
    end